crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -199,17 +204,33 @@ def network_checks
         @network_checks ||= ::Crowbar::Sanity.check
       end
 
+      def ceph_status
+        @ceph_status ||= Api::Crowbar.ceph_status
+      end
+
       def ceph_healthy?
-        Api::Crowbar.ceph_healthy?
+        ceph_status.empty?
+      end
+
+      def ha_presence_status
+        @ha_presence_status ||= Api::Pacemaker.ha_presence_check
```

Comment:
I don't remember here, so pacemaker is always installed? meaning `Api::Pacemaker` is always available?